training/index.md

```diff
@@ -21,19 +21,13 @@ sure we're all doing it consistently.
 ## Introduction Index
 This is a list of all the documents in this series. Start at the top and work your way down.
 
- - [What's Charcoal?][1]
- - [What's spam?][2]
- - [Privileges][3]
- - [Handling reports][4]
- - [Autoflagging][5]
- - [Further privileges: Core][7]
- - [Further privileges: Blacklister][6]
+1. [What's Charcoal?][/training/charcoal]

```

Comment:
Do square brackets work like round ones? I would expect [What's Charcoal?](/training/charcoal) and similarly for the following entries.

Reply:
It may depend on the markdown parser but I wouldn't expect it to work. 

https://www.markdownguide.org/basic-syntax/#links
https://commonmark.org/help/tutorial/07-links.html

It's either inline as `[text](link)` or with an ID as:

```
[text][1]

  [1]: link
```

Even if the square brackets do work, I'd stick to the widely-established format.